Tools/ardupilotwaf/boards.py

```diff
@@ -152,6 +153,8 @@ def configure_env(self, cfg, env):
                 '-g',
                 '-O0',
             ]
+        elif self.optimize is not None:
+            env.CXXFLAGS += [self.optimize]
```

Comment:
Nitpicking (but making it consistent with the rest of the file): make it multiple line:

```
env.CXXFLAGS += [
    self.optimize
]
```

I'd actually prefer `self.optimize` to be a dict itself, but I'm not fussed about that.